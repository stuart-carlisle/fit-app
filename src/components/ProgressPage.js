import React from 'react'
import ProgressFilters from './ProgressFilters'
import ChartView from './ChartView'

const ProgressPage = () => {

    return (
            <div>
                <ProgressFilters />
                <ChartView />
            </div>
    )   
}

export default ProgressPage