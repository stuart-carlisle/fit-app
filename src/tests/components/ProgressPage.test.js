import { shallow } from 'enzyme'
import React from 'react'
import ProgressPage from '../../components/ProgressPage'
 
test('Should render the progress page correctly',()=>{
    const wrapper = shallow(<ProgressPage />)
    expect(wrapper).toMatchSnapshot()
})