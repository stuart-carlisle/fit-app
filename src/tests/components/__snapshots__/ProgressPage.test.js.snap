// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the progress page correctly 1`] = `
<div>
  <Connect(ProgressFilters) />
  <Connect(ChartView) />
</div>
`;
